### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by harrisosserman**
@indutny

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,9 @@ const fft = new FFT(size);
 
 NOTE: `size` MUST be a power of two and MUST be bigger than 1.
 
+If you are looking to find the nearest power of 2 given the size of your dataset, here is a good tutorial:
```
Comment (indutny) [inline README.md]: Nit. Could be `[good tutorial](https://stackoverflow.com/questions/466204/rounding-up-to-next-power-of-2/466256#466256)`.

**Review by indutny — CHANGES_REQUESTED** [review]
LGTM, except for one nit!

Thank you

**Comment by harrisosserman** [comment]
@indutny updated!

**Comment by indutny** [comment]
Thank you! Landed.

**Pull request merged**